util/missingcrypto.txt

```diff
@@ -326,6 +326,7 @@ CMS_add_smimecap(3)
 CMS_add_standard_smimecap(3)
 CMS_data(3)
 CMS_dataFinal(3)
+CMS_dataFinal_digest(3)
```

Comment:
Unfortunately we do not allow adding undocumented public functions - that is by policy. There are two options - either make the function internal, or document it.

Reply:
OK. I've made it internal and renamed it to ossl_cms_DataFinal.

Reply:
> 

✅